### ISSUE TITLE
Possible Superfluous Value in Root Group Attributes

### ISSUE THREAD

**Issue by hasel001** [iho-ohi/S-102-Product-Specification#117]
Table Root Group Attributes (Section entitled Data Product Format, etc., which Metanorma numbers 11), Row Vertical Coordinate System, Column Remarks states (in relevant part):

Allowed values:
*6498 (Depth—​metres—​orientation down)
*6499 (Height—​metres—​orientation up)

My understanding is that we only ever allow the convention of positive down (i.e., 6498).

Am I missing something, or is it okay to delete the line listing 6499 as an option (and to change _values_ to _value_)?

**Comment by RohdeBSH** [comment]
Yes, a tiny detail.
If you choose the S-100 code 24 (Local Datum) for `verticalDatum`, the orientation of the Z-axis can be different from 6498 (orientation down).

**Comment by hasel001** [comment]
By allowing the choice of a system whose convention is orientation down, we seem to be creating a conflict.

1. In the paragraph directly following *Figure - S-102 Grid point location*, it says "Drying heights (drying soundings) are indicated by a negative depth value." The implication, in that case, is an orientation down configuration.

2. Table Root Group Attributes, Row Vertical Coordinate System, Column Remarks states (in relevant part):

Allowed values:
*6498 (Depth—​metres—​orientation down)
*6499 (Height—​metres—​orientation up)

3. Table BathymetryCoverage feature attribute parameters (in final table of annex), Row 4 says, "Drying heights (drying depths) are indicated by a negative value.

If (2) only allows 6498, (1) and (3) present no conflict. However, if (2) allows both, (1) and (3) will need to be rewritten to say something like: "Drying heights are indicated by having the opposite sign with respect to depths."

Do I infer correctly that (1) and (3) need to be changed in that way?

**Comment by RohdeBSH** [comment]
You're right, it does lead to a conflict.

Just prohibiting it on the side now also feels wrong somehow.
The attribute "Depth" is defined in the IHO registry as "The vertical distance from a given water level to the bottom.". So this fits both 6498 (Depth-metres-orientation down) and 6499 (Height-metres-orientation up).
However, the value range is limited from -14.0 to 11050.0.
This means that if you specify 6499 (Height-metres-orientation up), you can only map water areas up to a depth of 14.0m.
That is also nonsense.
So it only makes sense to remove the 6499 (Height-metres-orientation up).

By the way.
In comparison with the IHO registry, the "lower bound" and "upper bound" of Group_F are incorrect for the depth. That should be "lower"=-14 and "upper"=11050, right?

**Comment by rmalyankar** [comment]
An alternative is to update the constraints in the registry.

**Comment by hasel001** [comment]
I apologize that I hadn't seen @rmalyankar 's comment before I committed my edits. However, I think it makes sense to modify ours rather than theirs. People should not be using S-102 to navigate more than 14 meters above the surface of the water or lower than the Challenger Deep. Regardless, I have made the changes [here](https://github.com/iho-ohi/S-102-Product-Specification/commit/ab4b58d5f6d02edf6c0b94d263e477f00ffa2bf2). Thank you both for your help!
